docs/sources/developers/http_api/sso-settings.md

```diff
@@ -151,6 +162,11 @@ Status Codes:
 
 Updates the SSO Settings for a provider.
 
+When you submit new settings for a provider via API,
+Grafana verifies whether the given settings are allowed and valid.
+If they are, then Grafana stores the settings in the database and reloads
+Grafana services with no need to restart the instance.
```

Comment:
I think it'd worth to add a note which says (we have the same note on the configuration pages: [example](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/github/#configure-github-authentication-client-using-the-grafana-ui)):
```
If you run Grafana in high availability mode, configuration changes may not get applied to all Grafana instances immediately. You may need to wait a few minutes for the configuration to propagate to all Grafana instances.
```